### ISSUE TITLE
The MAE of ld50 dataset and the hyperparameters of the model

### ISSUE THREAD

**Issue by DreamMemory001** [mims-harvard/TDC#134]
**Describe your question.**

Hello, I want to ask a problem in your paper.  The metric of MAE is mean absolute error, why the dataset of ld50 is show the bigger the better in the paper? In addition, whether you add some tricks in the model training? If you not, maybe you can consider adding some tricks, likes DYRelu, CosineAnnealingLR and so on.  And in some datasets, I can`t reach the result level, or related with the model hyperparameters?

![e920d71dfaa9054d1f9a4cc692f1719](https://user-images.githubusercontent.com/32425458/149286524-3775fac1-8017-4f90-abd6-d2d19bacaa94.png)

**Comment by kexinhuang12345** [comment]
Hi thank you for spotting this, this is an typo. We would fix it in the next version. Also regarding the model, we use the basic hyperparameters as described in https://github.com/mims-harvard/TDC/tree/main/examples/single_pred/admet if you find a better solution, we suggest you submit to our leaderboard. Thanks!
